flink-connectors/flink-connector-elasticsearch6/src/main/java/org/apache/flink/streaming/connectors/elasticsearch6/Elasticsearch6ApiCallBridge.java

```diff
@@ -65,12 +66,16 @@
 	}
 
 	@Override
-	public RestHighLevelClient createClient(Map<String, String> clientConfig) {
+	public RestHighLevelClient createClient(Map<String, String> clientConfig) throws IOException {
 		RestClientBuilder builder = RestClient.builder(httpHosts.toArray(new HttpHost[httpHosts.size()]));
 		restClientFactory.configureRestClientBuilder(builder);
 
 		RestHighLevelClient rhlClient = new RestHighLevelClient(builder);
 
+		if (!rhlClient.ping()) {
```

Comment:
Should we add some logging here for information that we perform a communication step?

Reply:
Good idea.